content/docs/replication/deployment/powerflex.md

```diff
@@ -36,15 +36,16 @@ Run the following commands to verify that everything is installed correctly:
   Pods should be `READY` and `RUNNING`
 * Check that the controller config map is properly populated
     ```shell
-    
```

Comment:
why is this line removed?

Reply:
For consistency and to avoid a leading line in the codeblock.